ts/Maps/Projection.ts

```diff
@@ -11,19 +11,19 @@
 'use strict';
 
 import type { GeoJSONGeometry, LonLatArray } from 'GeoJSON';
-import type ProjectionDefinition from './ProjectionTypes';
-import type ProjectionOptions from 'ProjectionOptions';
+import type { ProjectionDefinition, ProjectionFunction } from './ProjectionTypes';
+import type { ProjectionOptions, ProjectionRotationOption } from 'ProjectionOptions';
 import type SVGPath from '../Core/Renderer/SVG/SVGPath';
 import H from '../Core/Globals.js';
 import registry from './Projections/ProjectionRegistry.js';
 import U from '../Core/Utilities.js';
 const {
     erase,
-    error,
     pick
 } = U;
 
 
+const deg2rad = Math.PI * 2 / 360;
```

Comment:
@TorsteinHonsi 
deg2rad is in Globals ->
https://github.com/highcharts/highcharts/blob/ee37092d0bfe5fdd9d45cf07d46aafa894e67e8b/ts/Core/Globals.ts#L144

Reply:
Yes. It's just so small and simple I am not sure it is worth the bother importing it.